.tugboat/steps/1-init.sh

```diff
@@ -74,7 +74,7 @@ nodejs -v | grep -q v$NODE_MAJOR
 shopt -s dotglob
 mkdir ../drainpipe-tmp
 mv * ../drainpipe-tmp/
-composer create-project drupal/recommended-project .
+composer create-project drupal/recommended-project:10.2.7 .
```

Comment:
This makes sense to me. Should it be `~10.2.7` so we get any patch releases of 10.2 in the future?

I think this is good regardless but wonder what @justafish thinks?